content/api/vision/coordinates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Coordinates
+---
+
+[Polar coordinates](https://en.wikipedia.org/wiki/Polar_coordinate_system) are represented by a distance from a set point and an angle from a set direction. [Cartesian coordinates](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) are ones that are defined using a unique set of values (typically _x_, _y_ and _z_) to specify a point in space.
+
+{{% notice note %}}
+All distances are measured in metres.
+{{% /notice %}}
+
+## Polar Coordinates
+The polar coordinates are taken from the position of the webcam.
+
+ - `distance_metres` - the distance from the camera to the marker.
+ - `rot_x_rad` / `rot_x_deg` - the angle of rotation in radians/degrees about the x axis. The x axis runs horizontally in front of the camera. A positive angle would be one where the marker is in front of the camera.
+ - `rot_y_rad` / `rot_y_deg` - the angle of rotation in radians/degrees about the y axis. The y axis runs vertically in front of the camera. A positive angle would be one where the marker is rotated clockwise about the y axis looking from the top of the marker.
```

Comment:
I think we should s/in front of/through/ in the phrase "The y axis runs vertically in front of the camera" since that would be more correct.

I also think we can clarify the directions by saying something like: "markers to the right of the camera will have a positive value, markers to the left will have a negative value".

Reply:
To me, _"The y axis runs vertically through the camera"_ sounds wrong?